doc/sphinx/source/recipes/recipe_kcs.rst

```diff
@@ -0,0 +1,141 @@
+.. _recipe_kcs:
+
+KNMI Climate Scenarios
+======================
+
+Description
+-----------
+
+This recipe implements the method described in `Lenderink et al., 2014 <https://doi.org/10.1088/1748-9326/9/11/115008>`_, to prepare the 2014 KNMI Climate Scenarios (KCS) for the Netherlands. A set of 8 global climate projections from EC-Earth were downscaled with the RACMO regional climate model. Since the EC-Earth ensemble is not readily representative for the spread in the full CMIP ensemble, this method recombines 5-year segments from the EC-Earth ensemble to obtain a large suite of "resamples". Subsequently, 8 new resamples are selected that cover the spread in CMIP much better than the original set.
+
+The original method created 16 resampled datasets:
+
+* 2 main scenarios: Moderate (M) and Warm (W) (Lenderink 2014 uses "G" instead of "M").
+* 2 'sub'scenarios: Relatively high (H) or low (L) changes in seasonal temperature and precipitation
+* 2 time horizons: Mid-century (MOC; 2050) and end-of-century (EOC; 2085)
+* 2 periods: Control (1981-2010) and future (variable)
+
+The configuration settings for these scenarios can be found in table 1 of Lenderink 2014's `supplementary data <https://iopscience.iop.org/1748-9326/9/11/115008/media/erl503687suppdata.pdf>`_.
+
+Implementation
+--------------
+
+The implementation is such that application to other datasets, regions, etc. is relatively straightforward. The description below focuses on the reference use case of Lenderink et al., 2014, where the target model was EC-Earth. An external set of EC-Earth data was used, for which 3D fields were available as well. In the recipe shipped with ESMValTool, however, the target model is CCSM4, so that it works out of the box with ESGF data only.
```

Comment:
Add why we need 3D data: "for which 3D fields for downscaling were available as well."